game/building/src/widgets/BuildPanel/widgets/Blocks/components/MaterialView/index.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import * as React from 'react';
+
+import {Material} from '../../lib/Material';
+
+export interface MaterialViewProps {
+  onClick: () => void;
+  selectedMaterial: Material;
+}
+
+export interface MaterialViewState {
+}
+
+class MaterialView extends React.Component<MaterialViewProps, MaterialViewState> {
+
+  constructor(props: MaterialViewProps) {
+    super(props);
+  }
+
+  render() {
+    if (this.props.selectedMaterial == null || this.props.selectedMaterial.icon==null)
```

Comment:
[blocks--braces](https://github.com/airbnb/javascript#blocks--braces) and by definition if else are multiline and must use braces.  

Also if return else is pointless.  https://github.com/airbnb/javascript#arrays--callback-return (second example)

if (expression) {
   return expression;
}
return else-expression;